sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -382,6 +396,58 @@ def _convert_span_to_envelope(span: Span) -> TelemetryItem:
         data.properties["_MS.links"] = json.dumps(links)
     return envelope
 
+def _convert_span_events_to_envelopes(span: Span) -> Sequence[TelemetryItem]:
+    envelopes = []
+    for event in span.events:
+        envelope = TelemetryItem(
+            name="",
+            instrumentation_key="",
+            tags=dict(_utils.azure_monitor_context),
+            time=ns_to_iso_str(event.timestamp),
+        )
+        # pylint: disable=protected-access
+        envelope.tags.update(_utils._populate_part_a_fields(span.resource))
+
+        envelope.tags["ai.operation.id"] = "{:032x}".format(span.context.trace_id)
+        if span.parent and span.parent.span_id:
+            envelope.tags["ai.operation.parentId"] = "{:016x}".format(
+                span.parent.span_id
+            )
+        properties = {}
+        if event.name == "exception":
```

Comment:
Is this name guaranteed to be this when using [record exception](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#record-exception)?

Reply:
Yes. https://github.com/open-telemetry/opentelemetry-python/blob/f81381cf8aca64a707d934f20c6c27d40b949dce/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L934